server/db/order.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require ('sequelize');
+const db = require('./_db');
+//import join here
+
+const Order = db.define('order', {
+  date: {
```

Comment:
should we clarify what type of date? purchaseDate maybe? Since we'll get a timestamp of createdAt from the database itself

Reply:
changed to purchaseDate